CityTour/mobile
# Feature/release apk

## Status
**READY**

## Migrations
NO

## Description
In order to generate an apk, Android requires that the application be signed before they can be installed, just have to generate a key and assign to the app. Then you can find your apk whit the name _app-released.apk_, inside the folder _android/app/build/outputs/apk/_

## Related PRs
 None


## Todos


## Deploy Notes
Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.

## Steps to Test or Reproduce

1. Git Pull
2. cd android && ./gradlew assembleRelease (_this generate the apk_)
3. react-native run-android --variant=release (_this should install it on your device_)

## Impacted Areas in Application
*folder app: 
 -  You got a new file named keystore
 -  Edit the file _build.gradle_, we add the lines to config the signing apk
*gradle.properties:
 - Adding the correct password and alias to the application
